examples/http/onvif_camera.json

```diff
@@ -0,0 +1,43 @@
+{
+    "logo": "onvif.svg",
+    "version": "1.0.0",
+    "name": "ONVIF camera monitoring",
+    "description": "Monitors the video capturing status of an ONVIF camera to check if capturing is still working or has stopped",
+    "category": "Hardware Integration",
+    "requirements": {
+        "credentials": false,
+        "sandbox_version": "1.12",
+        "others": [
+            "ONVIF"      
+        ]
+    },
+    "tags": [
+        "camera",
+        "onvif",
+        "video",
+        "streaming",
+        "status"
+    ],
+    "sample_period_s": 1800,
+    "has_actions": false,
+    "expected_variables": {
+        "table": "[1]"
+    },
+    "protocols": [
+        "TELNET",
```

Comment:
This is not correct, it is not the telnet protocol that you need and are using but the telnet library that is over TCP. Technically you are using the RTSP port with the telnet library.

Also HTTP is not fully correct.

Please change to a single protocol "ONVIF"